packages/core/src/utils/objectCheck.ts

```diff
@@ -0,0 +1,44 @@
+import type { ProofAttributeInfo } from '../modules/proofs'
+
+export class ObjectCheck {
```

Comment:
Maybe we can just make a method of this instead of a class?

Reply:
DONE